File: MvvmSample/app/src/main/java/com/abben/mvvmsample/binding/BindingApapters.java
package com.abben.mvvmsample.binding;

//import android.databinding.BindingAdapter;
import androidx.databinding.BindingAdapter;
import android.view.View;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

/**
 * Created by abben on 2017/10/27.
 */
public class BindingApapters {

    @BindingAdapter("visibleGone")
    public static void showHide(View view, boolean show) {
        view.setVisibility(show ? View.VISIBLE : View.GONE);
    }

    @BindingAdapter("imageUrl")
    public static void bingImage(ImageView imageView, String url){
        Glide.with(imageView.getContext()).load(url).into(imageView);
    }

}

File: MvvmSampleJetpack/app/src/main/java/com/infun/mvvmsamplejetpack/binding/BindingApapters.kt
package com.infun.mvvmsamplejetpack.binding

import android.view.View
import android.view.View.GONE
import android.view.View.VISIBLE
import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.bumptech.glide.Glide

/**
 *  @author abben
 *  on 2019/1/17
 */
class BindingApapters {

    @BindingAdapter("visibleGone")
    fun View.showHide(show: Boolean) {
        visibility = if (show) VISIBLE else GONE
    }

    @BindingAdapter("imageUrl")
    fun ImageView.bingImage(url: String) {
        Glide.with(context).load(url).into(this)
    }

}